pkg/controller/nodeipam/ipam/cloud_cidr_allocator.go

```diff
@@ -416,6 +419,20 @@ func (ca *cloudCIDRAllocator) updateMultiNetworkAnnotations(node *v1.Node, north
 	if node.Annotations == nil {
 		node.Annotations = make(map[string]string)
 	}
+
+	if ann, exists := node.Annotations[networkv1.NorthInterfacesAnnotationKey]; exists {
```

Comment:
this is confusing, the Line436 us just adding the value of this annotation

Reply:
I meant to use the MultiNetworkAnnotationKey, my mistake.

Reply:
I think the north interface annotation is better since it works for high performance networks as well.